app/src/main/java/it/quip/android/actvitiy/LoginActivity.java

```diff
@@ -3,30 +3,61 @@
 import android.content.Intent;
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
-import android.view.View;
-import android.widget.Button;
+import android.widget.Toast;
+
+import com.facebook.CallbackManager;
+import com.facebook.FacebookCallback;
+import com.facebook.FacebookException;
+import com.facebook.FacebookSdk;
+import com.facebook.login.LoginResult;
+import com.facebook.login.widget.LoginButton;
 
 import it.quip.android.R;
+import it.quip.android.network.FacebookClient;
 
 public class LoginActivity extends AppCompatActivity {
 
-    // TODO: remove this and all associated fake login code
-    Button btnLogin;
+    private CallbackManager mCallbackManager;
+    private LoginButton mBtnLogin;
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_login);
+    private void setupFacebookAuth() {
+        FacebookSdk.sdkInitialize(getApplicationContext());
```

Comment:
i was considering that initially but wanted to keep all Facebook shit encapsulated in a single method. I'll move it there though if we're doing that for parse already.